src/renderer/nav/Account.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+// import { useDispatch } from 'react-redux'
+
+import Stack from 'react-bootstrap/Stack';
+import { Button, Col, Row } from 'react-bootstrap';
+import ButtonGroup from 'react-bootstrap/ButtonGroup';
+import ButtonToolbar from 'react-bootstrap/ButtonToolbar';
+
+import AccountView from '../components/AccountView';
+import ProgramChangeView from '../components/ProgramChangeView';
+import LogView from '../components/LogView';
+import { useAppSelector, useAppDispatch } from '../hooks';
+import { selectValidatorNetworkState } from '../data/ValidatorNetwork/validatornetworkSlice';
+import {
+  accountsActions,
+  selectAccountsListState,
+} from '../data/SelectedAccountsList/selectedaccountsSlice';
+import { getAccount } from '../data/accounts/getAccount';
+
+function Account() {
+  const dispatch = useAppDispatch();
+
+  const validator = useAppSelector(selectValidatorNetworkState);
```

Comment:
i kinda like to group all the `useFoo`s together, but eh

Reply:
yeah, i'm still struggling with what to group and what not, its way too arbitrary.